src/bot.py

```diff
@@ -80,7 +75,16 @@
     logging.info("Using file variables.")
     try:
         config = configparser.ConfigParser()
-        config.read('config.txt')
+        config.read(config_path)
+
+        mininterval = int(config['intervals']['mininterval'])
+        maxinterval = int(config['intervals']['maxinterval'])
+
+        cooldown = randint(mininterval, maxinterval) * 24 * 60 * 60
+        # converts the random value from days to seconds
+
+        limit = int(config['formatting']['lines']) * int(config['formatting']['limit_per_line'])
+
        	CONSUMER_KEY = config['credentials']['CONSUMER_KEY']

```

Comment:
Have you really tested the bot? There are a few typos left in the source code that break it upon running.

Reply:
Where's the typo?

Reply:
Isn't there an indentation error right there? Maybe I'm wrong, but this really doesn't feel right from my end.

![image](https://user-images.githubusercontent.com/30193966/94832590-9d8ffa80-03fd-11eb-8fda-3121b7e00f89.png)

Reply:
![image](https://user-images.githubusercontent.com/38695526/94832753-9b826780-040e-11eb-947e-a518eae5fbc6.png)
I am not seeing it on VSCode.

Reply:
I'll confirm that in a bit, apologies.